### PR TITLE
feat: add vibrancy sidebar for macos

### PR DESCRIPTION
**Pull request by shatyuka**
## Description
Add vibrancy sidebar for macOS

### Checklist:

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you linted your code locally prior to submission?
* [x] Have you successfully ran app with your changes locally?

### CODE REVIEW

File: src/renderer/components/Aside/Index.vue [hunk 1]
```diff
@@ -59,6 +59,18 @@
           console.log(err)
         })
       }
+    },
+    data: function () {
```
Comment (agalwood) [inline src/renderer/components/Aside/Index.vue:63]: I think the computed attribute can be used to replace data here.
```javascript
vibrancy () {
  return is.macOS()
    ? {
      backgroundColor: 'transparent'
    }
    : {}
}
```

**Review by agalwood — CHANGES_REQUESTED** [review]
Please adjust the code : D

**Comment by agalwood** [comment]
👍 It is a cool feature !

File: src/renderer/components/Aside/Index.vue (hunk 1 above)
Comment (shatyuka) [reply src/renderer/components/Aside/Index.vue:63]: OK, I'm not familiar with vue. XD

**Review by agalwood — APPROVED** [review]

**Pull request merged**